contrib/win32/openssh/OpenSSHUtils.psm1

```diff
@@ -260,6 +262,42 @@ function Repair-UserSshConfigPermission
         Repair-FilePermission -Owners $UserSid,$adminsSid,$systemSid -AnyAccessOK $UserSid @psBoundParameters
 }
 
+<#
+    .Synopsis
+    Repair-SSHFolderPermission
+    Repair the file owner and permission of ssh folder & any files inside it
+#>
+function Repair-SSHFolderPermission
+{
+    [CmdletBinding(SupportsShouldProcess=$true, ConfirmImpact="High")]
+    param (
+        [parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]        
+        [string]$sshProgDataPath)
+
+    # SSH Folder - owner: System or Admins; full access: System, Admins; read or readandexecute/synchronize permissible: Authenticated Users
+    Repair-FilePermission -FilePath $sshProgDataPath -Owners $adminsSid, $systemSid -FullAccessNeeded $adminsSid,$systemSid -ReadAndExecuteAccessOK $authenticatedUserSid
+    # Files in SSH Folder (excluding private key & log files) 
+    # owner: System or Admins; full access: System, Admins; read/readandexecute/synchronize permissable: Authenticated Users
+    $privateKeyFiles = @("ssh_host_dsa_key", "ssh_host_ecdsa_key", "ssh_host_ed25519_key", "ssh_host_rsa_key")
+    Get-ChildItem -Path (Join-Path $sshProgDataPath '*') -Recurse -Exclude ($privateKeyFiles + "*.log") -File -Force | ForEach-Object {
```

Comment:
Lines 291 to 298 can be removed by removing -Exclude ($privateKeyFiles + "*.log")?